src/test/java/hw5/pages/HomePage.java

```diff
@@ -0,0 +1,141 @@
+package hw5.pages;
+
+import hw5.pages.components.HeaderMenu;
+import hw5.pages.components.LeftSideMenu;
+import io.qameta.allure.Step;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class HomePage extends BaseElements {
+
+    private WebDriver driver;
+
+    private LeftSideMenu leftSideMenu;
+    private HeaderMenu headerMenu;
+
+    private final static String url = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+        leftSideMenu = new LeftSideMenu(driver);
+        headerMenu = new HeaderMenu(driver);
+    }
```

Comment:
What is the proper place for the constructor according to Java Code Convention?

Reply:
After Instance variables. Have moved to the proper place.